sockeye/utils.py

```diff
@@ -700,12 +702,49 @@ def cleanup_params_files(output_folder: str, max_to_keep: int, checkpoint: int,
     """
     if max_to_keep <= 0:
         return
+
+    #
+    # make sure we keep N best params files from .metrics file according to strategy.
+    #
+
+    top_n = []
+    metrics_path = os.path.join(output_folder, C.METRICS_NAME)
+    
+    if max_params_files_to_cache > 0 and os.path.exists(metrics_path):
+        metric = cache_metric
+        strategy = cache_strategy
+            
+        maximize = C.METRIC_MAXIMIZE[metric]
+        points = get_validation_metric_points(model_path=output_folder, metric=metric)
+
+        if strategy == "best":
```

Comment:
Can we use the string constants instead?

Reply:
yes. Will address.